client/components/gsuite/gsuite-new-user-list/index.tsx

```diff
@@ -57,6 +61,13 @@ const GSuiteNewUserList: FunctionComponent< Props > = ( {
 	};
 
 	const onUserAdd = () => {
+		if ( context === 'inbox-management' ) {
```

Comment:
It may be simpler to just use `inbox` as the context. Also since we are using this string all over the place, we should establish a constant for it to reduce mistakes and aid refactorings.